forge/pyproject.toml

```diff
@@ -54,7 +54,7 @@ tiktoken = ">=0.7.0,<1.0.0"
 toml = "^0.10.2"
 uvicorn = { extras = ["standard"], version = ">=0.23.2,<1" }
 watchdog = "4.0.0"
-webdriver-manager = "^4.0.1"
+webdriver-manager = "^4.0.2"
```

Comment:
Was this needed?

Reply:
Thanks for the review @kcze.
This was done to address the [chromedriver](https://github.com/Significant-Gravitas/AutoGPT/pull/7694#issuecomment-2271928384) issue.